### ISSUE TITLE
Too much energy use in Mac OS even in idle

### ISSUE THREAD

**Issue by Overb00ffer** [jeffvli/sonixd#281]
On version 0.14 downloaded and installed with dmg the cpu usage is high even with no playable content it drains my laptop battery too fast :) In idle:
<img width="546" alt="sonixd_energy_ussage" src="https://user-images.githubusercontent.com/102738123/161059016-59db3047-d23e-406e-bd79-8bbdf52b5e3f.png">
While playing:
<img width="867" alt="Captura de Pantalla 2022-03-31 a las 14 54 06" src="https://user-images.githubusercontent.com/102738123/161059311-abcee77e-f88d-4dc0-a930-5f6544455682.png">
Thanks in advanced

**Comment by jeffvli** [comment]
If I'm understanding correctly, the energy usage is the same during idle as when playing?

I've done a few optimizations regarding CPU usage in the past such as #145, so I'm not sure what else may be causing it. I'll keep an eye out and see if anything else can be improved.

**Comment by Overbuffer** [comment]
No, sorry if the screenshot are not clear. In idle the process "Sonixd Helper (rendered)" is about 5-7 and goes top to 11,5 when is playing.

I have checked another projects with Electron like Element and there is no such consumption. So, like in #145  definitely some component must be yet running all the time :(
